### ISSUE TITLE
Question about seg2explorer

### ISSUE THREAD

**Issue by quail768** [EliHei2/segger_dev#45]
I'd like to see my data in the xeniumexplorer since you guys have already made the function(very cool!)
Is the function working or is it still in development?

Couple of questions:

What is the seg.df argment for the seg2explorer function?
Is the source_path argument for seg2explorer the initial xenium experiment directory before segmentation with segger?

Thanks!

**Comment by EliHei2** [comment]
Hi @quail768, thanks for making a deep dive into segger :) seg2explorer currently works on the previous version of file formats which is .csv.gz, I'm gonna update it for parquet files and notify you, asap. In the meanwhile, you might wanna look into https://github.com/quentinblampey/spatialdata_xenium_explorer from @quentinblampey.

**Comment by EliHei2** [comment]
@quail768 you're right next versions will include spatialdata output with help from @LucaMarconato. give us a couple of days to write the right funciton for the xenium explorer. re `find_markers` great catch! would you like to make a PR for the fix?
